### PR TITLE
Add test

### PR DESCRIPTION
**Pull request by AlicanC**
Added a very basic test to have some progress on #381

### CODE REVIEW

**Comment by trepidity** [comment]
Ooooh Cool!

Here's what I tried, ye, ney?

```
➜ yarn
yarn install v1.4.1-20180119.1606
info No lockfile found.
[1/4] 🔍  Resolving packages...
warning react-native > connect@2.30.2: connect 2.x series is deprecated
warning react-native > fbjs-scripts > gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning " > react-native@0.52.3" has unmet peer dependency "react@16.2.0".
[4/4] 📃  Building fresh packages...
success Saved lockfile.
✨  Done in 16.21s.
```

```
➜ yarn test
yarn run v1.4.1-20180119.1606
$ jest
 FAIL  ./sound.test.js
  ✕ it can create sound (29ms)

  ● it can create sound

    TypeError: this._nativeModule.addListener is not a function

      42 |
      43 |     if (!IsWindows) {
    > 44 |       this.onPlaySubscription = eventEmitter.addListener(
      45 |         'onPlayChange',
      46 |         (param) => {
      47 |           const { isPlaying, playerKey } = param;

      at NativeEventEmitter.addListener (node_modules/react-native/Libraries/EventEmitter/NativeEventEmitter.js:48:26)
      at Sound.registerOnPlay (sound.js:44:46)
      at sound.js:81:12
      at Timeout._onTimeout (sound.test.js:12:22)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:        2.843s
Ran all test suites.
error An unexpected error occurred: "Command failed.\nExit code: 1\nCommand: sh\nArguments: -c jest\nDirectory: /Users/jjennings/projects/react-native-sound\nOutput:\n".
info If you think this is a bug, please open a bug report with the information provided in "/Users/jjennings/projects/react-native-sound/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

**Comment by AlicanC** [comment]
I don't know why it failed on your side. Changed a few things that might fix the issue, but I'm not sure.

File: sound.test.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  },
+  play(key, callback) {
+    const successful = true;
+    setTimeout(() => callback(successful));
+  },
+  addListener() {
+    return {
+      remove() {
+
+      };
```
Comment (trepidity) [inline sound.test.js]: this semicolon breaks it :)

**Comment by trepidity** [comment]
How about also moving the test into a tests folder?

**Comment by AlicanC** [comment]
Updated PR.

**Comment by paulmelnikow** [comment]
Hi! This branch seems no longer to have the tests into. Would it be possible to re-push them, either where or in a new PR?

**Comment by paulmelnikow** [comment]
If you're able to re-open this, that would be great!

**Pull request closed without merging**